.github/workflows/c.yml

```
@@ -29,6 +29,23 @@ jobs:
       # only run if files in `.docker/c/` unchanged
       should-run: ${{ steps.changes.outputs.docker-c == 'false' }}
 
+  report-status-tests:
+    runs-on: ubuntu-latest
+    needs: [diff, diff-header-only-ml-dsa, diff-header-only-ml-kem, build, build-header-only-ml-dsa, build-header-only-ml-kem]
```

Putting this underneath may be a little more readable
```suggestion
    needs:
      - diff
      - diff-header-only-ml-dsa
      - diff-header-only-ml-kem
      - build
      - build-header-only-ml-dsa
      - build-header-only-ml-kem
```